docs/theory/diffusion/intro.rst

```diff
@@ -0,0 +1,140 @@
+Introduction to Diffusion analysis
+==================================
+
+This chapter gives an introduction to diffusion and the recommended methods for diffusion analysis available in Pylake. 
+Further, this page summarizes important steps for analyzing diffusive tracks, such as localization refinement. 
+This chapter does not introduce the math or syntax, but rather the underlying thoughts, providing an introduction for anyone who wants to start analyzing a kymograph with a diffusive protein.
+
+
+Target search
+----------------------------
+
+Proteins diffusive through a cell and often have to search for a target on DNA. A long-standing question in biology is how proteins find their target on the huge genome through random, diffusive motion. 
+The first hypothesis was that protein diffused through the cytoplasm (in 3D), until they found their target. Berg and von Hippel were the first to demonstrate that the Lac repressor
+finds its target faster than can be explained with 3D diffusion alone. 
+Now, many experiments have shown that proteins often diffuse both in 1D diffusion (along DNA) and 3D diffusion (in the cytoplasm), also called facilitated diffusion :cite:`Berg1976, Berg1981`, which is more efficient that 3D diffusion alone.  
+
+.. image:: figures/target_search.png
+
+1D diffusion can can be directly visualized by holding a force-stretched DNA between optical tweezers and observing a fluorescently labeled protein bind and move along this DNA tether. 
+Pylake has many functions implemented for analyzing data from such an assay. This chapter focusses on the analysis of 1D diffusion of a protein on DNA.
+
+.. image:: figures/1D_diffusion.png
+
+
+
+The diffusion coefficient
+---------------------------
+
+A purely diffusive protein has the same probability to step to the right as to the left from any location on the DNA, therefore the average displacement is zero. 
+Rather than through velocity, diffusion is therefore quantified through the *diffusion coefficient (D)*. The diffusion coefficient quantifies how fast a protein explores its environment; 
```

Comment:
```suggestion
Since diffusion is distinct from directed motion, which would be characterized by a velocity, it is quantified with a different quantity named the *diffusion coefficient (D)*. The diffusion coefficient quantifies how fast a protein explores its environment; 
```